packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -132,6 +133,10 @@ class DraggableScrollableSheet extends StatefulWidget {
   /// The initial fractional value of the parent container's height to use when
   /// displaying the widget.
   ///
+  /// Rebuilding the sheet with a new [initialChildSize] will only move the
+  /// the sheet to the new value if the sheet has not yet been dragged since it
+  /// was first built or since the last call to [DraggableScrollableActuator.reset].
+  ///
```

Comment:
This seemed like the most intuitive/desirable behavior, but I'd love some input here as this was just a judgement call.

Also see documentation changes for `snap` and `snapSizes` immediately below.